plugin/sampling/strategystore/adaptive/processor.go

```diff
@@ -310,12 +310,26 @@ func (p *Processor) aggregateThroughput(throughputs []*model.Throughput) service
 			t.Count += throughput.Count
 			t.Probabilities = merge(t.Probabilities, throughput.Probabilities)
 		} else {
-			aggregatedThroughput[service][operation] = throughput
```

Comment:
This is probably a bug (unintended pointer copy makes so that the input argument `throughputs` is changed during this method execution)

Reply:
bug hunted thanks to `shuffle` :yum: